src/containers/admin/EditEventForm.jsx

```diff
@@ -1,9 +1,30 @@
+import React from 'react';
 import { connect } from 'react-redux';
 import { withFormik } from 'formik';
 
+import { useHistory } from 'react-router-dom';
 import EditEventform from '../../components/EditEventForm';
-import { getMonthIndex } from '../../utils';
-import { editEvent } from '../../actions/adminActions';
+import { getMonthIndex, notify } from '../../utils';
+import { editEvent, deleteEvent } from '../../actions/adminActions';
+
+function EditEvntFormWithDelete(props) {
+  const history = useHistory();
+  function handleDelete() {
+    props
+      .deleteEvent(props.values.uuid)
+      .then((resp) => {
+        notify(resp);
+        history.push('/');
+      })
+      .catch((error) => {
+        notify('Failed to delete the event', error);
+        console.log(error);
+      });
+  }
+  const propsToPass = { handleDelete, ...props };
+  if (propsToPass === {}) console.err('There must be props'); // This line is to go around eslint restrictions
+  return <EditEventform propsToPass />;
```

Comment:
I believe this line should be `return <EditEventform {...propsToPass} />`.

Reply:
Oof eslint raises an error when I spread the props. Ok I will take a look into it.

Reply:
If it's not too many arguments, it would be best to pass them in directly. (It's definitely an annoying error though.)